src/Comments.tsx

```diff
@@ -0,0 +1,30 @@
+import * as Types from './types';
```

Comment:
Currently when new comments are added, there are some rendering issues.

I think the CSS for this section needs to be made more elastic.

Reply:
Let's go through this part in a pair programming session

Reply:
I appreciate that! I actually already fixed it